tasks/cucumber.js

```diff
@@ -184,10 +184,8 @@ module.exports = function(grunt) {
         handler(grunt, options, commands, function handlerCallback(err) {
             if (err) {
                 grunt.log.error('failed tests, please see the output');
```

Comment:
Invoking done(false) causes the grunt task to report a failure, which probably isn't the desired intention here - failed tests should not result in a failed grunt task.

The issue with this is that any subsequent grunt tasks do not run. In our case, our grunt command sequence looked like: 
```['mockserver_start', 'cucumber', 'mockserver_stop']```

In this case, if a test case failed, mockserver_stop was never invoked.

The expected behavior is that the task should succeed and failures should be dealt with by inspecting the test results in a subsequent step / process.